engine/crates/parser-graphql/src/metadata.rs

```diff
@@ -0,0 +1,11 @@
+use engine::registry::ConnectorHeaders;
+use url::Url;
+
+pub struct GraphQlConnectorMetadata<'a> {
+    pub(crate) name: &'a str,
+    pub(crate) namespace: bool,
```

Comment:
The explanation from your PR description would be useful on this field. This struct is about the connector arguments if I understand this right, so maybe this is already documented elsewhere, then ignore the suggestion.

Reply:
Oh ok, I didn't notice it wasn't used.